sunpy/net/scraper.py

```diff
@@ -351,83 +266,37 @@ def _ftpfileslist(self, timerange):
     def _localfilelist(self, timerange):
         pattern = self.pattern
         pattern_temp = pattern.replace('file://', '')
+        timepattern = self.timepattern
+        timepattern_temp = timepattern.replace('file://', '')
```

Comment:
Is there no cleaner way than to define temp vars like this for pattern and timepattern?